middle_end/flambda2/to_cmm/to_cmm_expr.ml

```diff
@@ -522,7 +589,7 @@ and let_cont_exn_handler env res k body vars handler ~catch_id arity =
         C.letin extra_param ~defining_expr:(C.var mut_var) ~body:handler)
```

Comment:
Please put a comment explaining why we don't count these expressions in the free variables of the handler

Reply:
comment added